README.md

```diff
@@ -1,106 +1,112 @@
-  make down
-```
-
-## Requirements
-
-To build and run:
-
-- docker
-- make >=4.2
-- awk, jq (optional tools within makefiles)
-
-To develop, in addition:
-
-- python 3.6 (this dependency will be deprecated soon)
-- nodejs for client part (this dependency will be deprecated soon)
-- [vscode] (highly recommended)
-
-This project works and is developed mainly under **linux** but, with some adjustments, it can also be done under windows (see notes below).
-
-##### Setup for **windows**
-
-In windows, it works under [WSL] (windows subsystem for linux). Some details on the setup:
-
-- [Install](https://chocolatey.org/docs/installation) [chocolatey] package manager
-  - ``choco install docker-for-windows``
-  - ``choco install wsl`` or using [instructions](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
--  Follow **all details** on [how to setup flawlessly](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly) docker for windows and [WSL]
-
-
-## Releases
-
-- [Git release workflow](ops/README.md)
-- Public [releases](https://github.com/ITISFoundation/osparc-simcore/releases)
-- Production in https://osparc.io
-- [Staging instructions](docs/staging-instructions.md)
-
-
-
-
-<!-- ADD REFERENCES BELOW AND KEEP THEM IN ALPHABETICAL ORDER -->
-[chocolatey]:https://chocolatey.org/
-[vscode]:https://code.visualstudio.com/
-[WSL]:https://docs.microsoft.com/en-us/windows/wsl/faq
+# osparc-simcore platform
+
+**WARNING** This application is **still under development**.
+
+<!-- NOTE: when branched replace `master` in urls -->
+[`master`](https://github.com/itisfoundation/osparc-simcore/tree/master)
+[![Requires.io]](https://requires.io/github/ITISFoundation/osparc-simcore/requirements/?branch=master "State of third party python dependencies")
+[![travis-ci]](https://travis-ci.org/ITISFoundation/osparc-simcore "State of CI: build, test and pushing images")
+[![coverals.io]](https://coveralls.io/github/ITISFoundation/osparc-simcore?branch=master "Test coverage")
+[![github.io]](https://itisfoundation.github.io/)
+[![itis.dockerhub]](https://hub.docker.com/u/itisfoundation)
+
+
+<!-- THIS SERVICE MALFUNCTIONS
+[![webserver]](https://microbadger.com/images/itisfoundation/webserver "More on itisfoundation/webserver:staging-latest image")
+[![director]](https://microbadger.com/images/itisfoundation/director "More on itisfoundation/director:staging-latest image")
+[![sidecar]](https://microbadger.com/images/itisfoundation/sidecar "More on itisfoundation/sidecar:staging-latest image")
+[![storage]](https://microbadger.com/images/itisfoundation/storage "More on itisfoundation/storage:staging-latest image")
+-->
+
+
+<!-- ADD HERE ALL BADGE URLS -->
+[Requires.io]:https://img.shields.io/requires/github/ITISFoundation/osparc-simcore.svg
+[travis-ci]:https://travis-ci.org/ITISFoundation/osparc-simcore.svg?branch=master
+[coverals.io]:https://coveralls.io/repos/github/ITISFoundation/osparc-simcore/badge.svg?branch=master
```

Comment:
this one is only contributed to by travis-ci. not sure whether it is correct.
we also have codecov.io which would be that and this is contributed to by github-ci:
[![codecov](https://codecov.io/gh/ITISFoundation/osparc-simcore/branch/master/graph/badge.svg)](https://codecov.io/gh/ITISFoundation/osparc-simcore)

Reply:
I took care of this in #1482